src/routes/routes.ts

```diff
@@ -1,7 +1,11 @@
-import { Router } from "express";
+import { NextFunction, Request, Response, Router } from "express";
 
 const router: Router = Router();
 
-router.get("/", (req, response) => response.send("This is the Strike Off Objections app"));
+const renderTemplate = (template: string) => (req: Request, res: Response, next: NextFunction) => {
+    return res.render(template, { templateName: template });
```

Comment:
Presume we're still using the same formatting standard as PTF - 2 spaces for indents? If so, the indent here will need amending.

Reply:
Ammended.